src/sqlite3_to_mysql/mysql_utils.py

```
@@ -124,6 +124,23 @@
+    mysql_version: Version = get_mysql_version(version_string)
+    if "mariadb" in version_string.lower():
+        return False
+    # Only MySQL 8.0.19 and later support VALUES alias
+    if mysql_version.major >= 8:
+        if mysql_version.major > 8:
+            return True
+        if mysql_version.minor > 0:
+            return True
+        if mysql_version.micro >= 19:
+            return True
+        return False
```

Ruff

Return the condition `mysql_version.micro >= 19` directly

Replace with `return mysql_version.micro >= 19`

(SIM103)